add bitwarden

Deivedux opened the pull request:
**Type of pull request:** product addition

**Related issues:** no issues, yet.

Deivedux committed:
add bitwarden

doamatto commented on products/bitwarden.toml:
```diff
@@ -0,0 +1,93 @@
+[rubric.behavioral-marketing]
+value = "yes-opt-out"
+citations = ["Bitwarden respects your email communications and marketing preferences. If you prefer not to receive product release notes communications or promotional email messages (such as product updates, security alerts, marketing, events, training and certifications) from Bitwarden, you can unsubscribe from Bitwarden email marketing by following the unsubscribe link located at the bottom of  ...
+notes = ["While their website prompts the optional cookie settings to all new visitors, email marketing is still enabled by default."]
```
> All of the categories of Personal Information we collect about you (as detailed above) are used for the following purposes:
>
>    Providing our services (for example, account servicing and maintenance, customer service, advertising and marketing, analytics, and communication about our services)

I'm curious if this is able to be optted-out from as well. The wording amkes it seem like the advertising and marketing may be a separate from that which is sent by e-mail.

doamatto commented on products/bitwarden.toml:
```diff
@@ -0,0 +1,93 @@
+
+[rubric.data-deletion]
+value = "yes-automated"
+citations = ["If you terminate your relationship with Bitwarden, we will delete your Personal Information in accordance with our data retention policies."]
```
Automated is technically true, but it's not in writing. What is in writing is e-mail :

> We enable you to access, correct, and delete your account with the Bitwarden Service at any time. If you would like to request assistance with accessing, correcting, or deleting your Personal Information, please submit your request to us by email at privacypolicy@bitwarden.com.

doamatto commented on products/bitwarden.toml:
```diff
@@ -0,0 +1,93 @@
+- If you interact with the Bitwarden Community or training, or registered for an exam or event, we may collect biographical information and the content that you share.
+- Information gathered via cookies, pixel tags, logs, or other similar technologies."""
+]
+notes = ["The list is difficult to argue to be exhaustive due to the use of \"such as\" when listing the collected data."]
```
That's fine — I'd remove the note because this is what is used in the definition of that value for this rubric item.

doamatto commented on products/bitwarden.toml:
```diff
@@ -0,0 +1,93 @@
+value = "yes"
+citations = ["If we make any material changes, we will notify you by email (sent to the email address specified in your account registered with the Site or Bitwarden Service) or by means of a notice on the Site or Service."]
+
+[rubric.security]
```
I think it's fair to note their HIPAA compliance and history of SOC2, SOC3, and other audits ([from here](https://bitwarden.com/compliance/)).

doamatto reviewed (changes requested):
Sorry for the wait — here's a few things that need changing.

doamatto committed:
fix(product): changes to Bitwarden

- Update data deletion score (yes-automated -> yes-contact)
- Update last modified
- Update data collected list to touch on data from the service and
  not just the website (bitwarden.com)
- Update security score (somewhat -> yes-independent-audits)
- Add subprocessors list and updated third-party-access score
- Remove superfluous note for data collected list
